models/agora_snapshot/optimism_proposals/agora_snapshot_for_against_proposals.sql

```diff
@@ -0,0 +1,277 @@
+-- ForAgainst Proposals Available On Agora And Snapshot Platform
+
+{{ config(tags=['dunesql']
+    ,alias = 'agora_snapshot_for_against_proposals'
+    ,materialized = 'incremental'
+    ,schema = 'agora_snapshot_optimism_proposals'

```

Comment:
it's probably a better idea to put it under `governance_optimism`?

Reply:
Created a governance folder under models, and removed the redundant bifurcation as pointed out earlier. 
Thanks for the suggestion. 
Let me know if we can make any more changes to the spell.